chromium_src/components/sync/base/model_type_unittest.cc

```diff
@@ -14,7 +14,7 @@ TEST_F(ModelTypeTest, EncryptableUserTypes) {
 
 // This test is supposed to fail when sync types are increased/decreased
 TEST_F(ModelTypeTest, ModelTypeCounts) {
-  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 54);
+  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 56);
```

Comment:
new types are
```
  // Segmentation data.
  SEGMENTATION,

  // A tab group saved by a user. Currently only supported on desktop platforms
  // (Linux, Mac, Windows, ChromeOS).
  SAVED_TAB_GROUP,
```
`SEGMENTATION` if for Google Analytics, it is not enabled for us even if we set `Sync everything` 
`SAVED_TAB_GROUP` appears to be synced through `Open tabs`, it does not require the additional switch , but it doesn't work for us.

Follow-up  issues https://github.com/brave/brave-browser/issues/27240, https://github.com/brave/brave-browser/issues/27241